src/scenes/cheatSheet.js

```diff
@@ -1,25 +1,42 @@
-let cheatSheetImg;
-let cheatSheetBtnImg;
-let cheatSheetBtn;
-const cheatSheet = {
-    "setup": function()
-    {
-		cheatSheetImg = loadImage("assets/textures/CheatSheet.png",
-			() => cheatSheetImg = scaleNearestNeighbor(cheatSheetImg, width, height));
-		cheatSheetBtnImg = loadImage("assets/textures/CheatSheetBackButton.png",
+import Scene from "../objects/scene.js";
+import { scaleNearestNeighbor } from "../helperFunctions/imageFunctions.js";
+import ImageButton from "../objects/imageButton.js";
+import sceneManager from "../objects/sceneManager.js";
+import { MAIN_MENU } from "../constants/sceneConstants.js";
+import canvasManager from "../objects/canvasManager.js";
+
+export default class CheatSheet extends Scene {
+	constructor() {
+		super();
+		this.cheatSheetImg = {};
+		this.cheatSheetBtnImg = {};
+		this.cheatSheetBtn = {};
```

Comment:
I tried my best to preserve the existing implementation of all of this, but I feel like we might be able to simplify here

Reply:
Part of it is going to be defining the callbacks as their own functions for if nothing, at least flatten out the callback hell in this file.